lib/sisimai/bite/email/exim.rb

```diff
@@ -0,0 +1,522 @@
+module Sisimai::Bite::Email
+  module Exim
+    # Sisimai::Bite::Email::Exim parses a bounce email which created by Exim.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/Exim.pm
+      require 'sisimai/bite/email'
+
+      ReE = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->